src/pages/VerifyCollab.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import {Button, Heading, HStack, Text, useToast, VStack} from "@chakra-ui/react";
+import {Form, Formik, FormikProps, FormikValues} from "formik";
+import {Icon} from "@chakra-ui/icons";
+import {ReactComponent as WindowIllustration} from "../assets/window.svg";
+import {PasswordField} from "../forms/AuthFields";
+import {LineBackground} from "../components/Backgrounds";
+import {baseSchema} from "../forms/Schemas";
+import useLocalStorage from "use-local-storage";
+import {useFetch} from "use-http";
+import {useNavigate, useParams} from "react-router-dom";
+
+export default function VerifyCollab() {
+    const { adminId } = useParams()
+    const navigate = useNavigate()
+    const schema = baseSchema.pick(['password', 'repeatPassword'])
+    const { put, response } = useFetch(`/api/admins/${adminId}/profile`)
+    const [adminData, setAdminData] = useLocalStorage<AdminProps | undefined>('adminData', undefined)
+    const toast = useToast()
+
+    const verifyCollab = async (values: FormikValues) => {
+        const fetchedData = await put(values)
```

Comment:
We want to use this component to also verify the collaborator's account, so we have 2 alternatives:

1. Add the field `private boolean verified` to the UserDTO in the backend, and then on line 22 here do the request like this: `await put({ verified: true, ...values })`
2. Keep the put request as is but after response.ok redirect the user from here to the `'/verify/:adminId'` endpoint - so changing to ```navigate(`/verify/${adminId}`)```

Option 2 is probably easier? This way you also don't need to have useLocalStorage(...) in this component at all